tests/installation/bootloader_zkvm.pm

```diff
@@ -71,8 +72,12 @@ sub run() {
     my $size_i = get_var('HDDSIZEGB') || '4';
 
     if (my $hdd = get_var('HDD_1')) {
+        my $hdd_dir  = "/var/lib/openqa/share/factory/hdd";
```

Comment:
"PRJDIR" : "/var/lib/openqa/share",
"PRODUCTDIR" : "/var/lib/openqa/share/tests/sle/products/sle",

looks fine for me